netbench-orchestrator/src/russula/netbench/server_worker.rs

```diff
@@ -0,0 +1,224 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::ServerContext;
+use crate::russula::states::StateApi;
+use crate::russula::states::TransitionStep;
+use crate::russula::{
+    error::{RussulaError, RussulaResult},
+    event::EventRecorder,
+    netbench::server_coord::CoordState,
+    network_utils::Msg,
+    workflow::WorkflowTrait,
+};
+use core::fmt::Debug;
+use serde::{Deserialize, Serialize};
+use std::{fs::File, net::SocketAddr, process::Command};
+use sysinfo::{Pid, PidExt, ProcessExt, SystemExt};
+use tokio::net::{TcpListener, TcpStream};
+use tracing::{debug, error, info};
+
+// Only used when creating a state variant for comparison
+const PLACEHOLDER_PID: u32 = 1000;
```

Comment:
Rather than using a bogus value, could we use an `Option`?

Reply:
This is serialized state so we will need to serialize some state or parse Option. I can create an issue for this but dont think we should explore this work for this PR